### DISCUSSION TITLE
Fill method

### DISCUSSION THREAD

**Discussion by x3x4x5** [SixLabors/ImageSharp#2105] [Ideas]
Hey,

I want to Fill bigger background image with smaller images. Fill method always makes squares, even if the brush image is a rectangle. It seems to take into account only the width of the image.

Code:
` img.Fill(new ImageBrush(frameImage), new RectangleF(Convert.ToInt32(transformation.AnchorX.Value + parentX),
                                                        Convert.ToInt32(transformation.AnchorY.Value + parentY),
                                                        Convert.ToInt32(parentSize.X2),
                                                        Convert.ToInt32(parentSize.Y2)));`

Source image (frameImage):
![source](https://user-images.githubusercontent.com/104835406/166448477-a3564159-9a6e-400e-a0a7-ab36256dfaa5.png)

Result:
![result](https://user-images.githubusercontent.com/104835406/166447946-94c273dc-cd1e-41e4-9219-2b8f6917409b.png)

**Comment by JimBobSquarePants** [comment]
Please do not use the feature suggestions discussion group for questions!

**Reply by JimBobSquarePants** [reply]
You’ve not even used the right repository!
